### PR TITLE
update response type for /auth endpoint

### PR DESCRIPTION
**Pull request by rolyatmax**
This makes updates in response to changes to the Jamsocket API's `/auth` endpoint.

This also does a check for the latest CLI version when an error is encountered.

### CODE REVIEW

File: src/commands/login.ts [hunk 1]
```diff
@@ -18,11 +18,12 @@ export default class Login extends Command {
...
     const config = readJamsocketConfig()
     if (config !== null) {
-      const { account, token } = config
+      const { token } = config
       const publicPortion = getTokenPublicPortion(token)
       try {
-        await api.checkAuth(token)
-        this.log(`You are already logged into account "${account}" with the token "${publicPortion}.********". To log into a different account, run jamsocket logout first.`)
+        const result = await api.checkAuth(token)
+        if (result.account === null) throw new Error('No account found for logged in user.')
```
Comment (paulgb) [inline src/commands/login.ts]: Maybe we should advise the user on what to do in this situation? E.g. go to [this URL] first to create an account?

File: src/commands/login.ts [hunk 2]
```diff
@@ -49,6 +50,7 @@ export default class Login extends Command {
     }
 
     const { account } = await api.checkAuth(token)
+    if (account === null) throw new Error('No account found for user.')
```
Comment (paulgb) [inline src/commands/login.ts]: ditto here

File: src/request.ts [hunk 3]
```diff
@@ -13,11 +13,43 @@ type RequestReturn = {
   headers: Headers;
 }
 
-const version = require('../package.json').version
+const version = '0.0.10' // require('../package.json').version
```
Comment (paulgb) [inline src/request.ts]: Why the change?

**Review by paulgb — APPROVED** [review]

File: src/request.ts (hunk 3 above)
Comment (rolyatmax) [reply src/request.ts]: Whoops! Was testing and accidentally left this in!

File: src/commands/login.ts (hunk 1 above)
Comment (rolyatmax) [reply src/commands/login.ts]: Yeah, good call. Updated
